geopandas/io/sql.py

```diff
@@ -245,16 +245,26 @@ def _get_srid_from_crs(gdf):
     srid = -1
```

Comment:
Due to the `try/except` wrapping the for loop, I am thinking it might be safer to set `srid = None` here and add this at the bottom:
```python
    if srid is None:
        srid = -1
        warnings.warn(warning_msg, UserWarning, stacklevel=2)
```

Which means the `else` that is part of the `for` loop would be removed as well ...

Reply:
That works too!

Reply:
Thanks for being patient and flexible :+1: